Helper/Discount.php

```diff
@@ -1293,6 +1293,9 @@ public function loadCouponCodeData($couponCode)
      */
     public function convertToBoltDiscountType($couponCode)
     {
+        if ($couponCode == "") {
+            return "";
+        }
```

Comment:
We probably need this case covered at Bolt's side for consistency. Meanwhile, perhaps some existing discount type should be returned instead of an empty string. `fixed_amount` maybe, cause this discounts behave similar to  (the same as?) coupons, except there's no coupon and it is auto-applied to cart when some conditions match.

Reply:
Yes, valuable comment. I changed to fixed_amount for now and wiil discuss with @melvin0008 if it's the best option.

Reply:
@melvin0008 confirmed that this is a correct way.